cms/templates/settings.html

```diff
@@ -356,6 +356,15 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                       a_link_end=HTML("</a>")
                     )}</span>
                 </li>
+                <li class="field text" id="field-course-about-sidebar-html">
+                    <label for="course-about-sidebar-html">${_("Course About Sidebar HTML")}</label>
+                    <textarea class="tinymce text-editor" id="course-about-sidebar-html"></textarea>
```

Comment:
@caesar2164 doubt this is causing the a11y error, but you want to add an ID to the span below and reference it via aria-describedby on the textarea element so that the "tip" is read out when the text area receives focus, and not afterwards, when the user is likely done using the input.

Reply:
Just curious, but can you see if you still get the error if you close the label element after the textarea so that textarea is a child of label.  That should not be required, but I am just curious.

Reply:
Ok just did both of these to see if it fixes the a11y error.